Text extraction with pandoc (#71)
dataiku/dss-plugin-tesseract-ocr: CHANGELOG.md

# Changelog

## [Version 2.1.0](https://github.com/dataiku/dss-plugin-tesseract-ocr/releases/tag/v2.1.0) - Feature release - 2023-06

> - Add a recipe to extract text directly (not OCR) from multiple types of files

Reviewer: ```suggestion
- Add a recipe to extract text content from various file types
```
